webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path');
+const glob = require('glob');
+
+const isProduction = process.env.NODE_ENV == "production";
+
+const config = {
+  entry: ["./custom/styles/styles.scss"],
+  output: {
+    path: path.resolve(__dirname, "./public/scripts"),
+    filename: 'index.js'
+  },
+  devServer: {
+    open: true,
+    host: "localhost",
+  },
```

Comment:
is this disabled by default in production? I couldn't find docs about that on a quick search

Reply:
I just removed this.  This is for webpack dev server which we're not using.